grails-app/conf/Config.groovy

```diff
@@ -106,13 +109,18 @@ scheduler = {
 }
 
 environments {
+    test {
+        grails.plugin.raven.active = false
+    }
     development {
         grails.logging.jul.usebridge = true
+        grails.plugin.raven.active = false
     }
     production {
         grails.logging.jul.usebridge = false
+        grails.plugin.raven.active = true
         quartz.autoStartup = true
-        grails.serverURL = "http://www.mama-ng-scheduler.com"
+        grails.serverURL = "http://www.scheduler.com"
```

Comment:
``www.scheduler.com`` is an actual website. Maybe we want ``scheduler.example.com``?